guide_to_writing_techreports.tex

```diff
@@ -175,8 +176,7 @@ \subsection{Font and spacing}
 
 \clearpage
 \subsection{Math}
-Mathematical symbols are treated as shown in the following example:
-% TODO: We should really have a reference for this style - maybe AMS?
+Mathematical symbols are typeset as shown in the following example \citep{ams}:
```

Comment:
They do sort of, see this [link](http://www.ams.org/arc/styleguide/AMSstyleguide.pdf). However, they number equations to the left and use parentheses in cross-references (Section 13.14 p. 112). Also for matrices, they only display parentheses instead of square brackets (Section 13.12 p. 110). Perhaps we should rather refer to the book [Mathematics in Text](http://www.ams.org/arc/styleguide/mit-2.pdf)? They use square brackets (Section 2.4.8d p. 26) and simply state that equation numbering must be consistent (Section 4.1.2b p. 50)